run/cli_ioproducer.go

```diff
@@ -0,0 +1,35 @@
+package run
+
+import (
+	"context"
+	"io"
+	"log"
+	"os"
+)
+
+// CliIOProducer is the IOProducer for the CliRunner. The input and output paths
+// are used to configure the input and output readers and writers. If the paths
+// are empty, os.Stdin and os.Stdout are used.
+func CliIOProducer(_ context.Context, cfg CLIRunnerConfig) IOData {
```

Comment:
Why not `CLIOProducer`? A bit less readable I admit, but consistent to the other naming. Your call 😅

Reply:
The linter was complaining about "unknown word", which isn't terrible, but that was one reason not to capitalize. The other reason was, as you say, readability. Any strong opinions? @ryanjoneil @merschformann @davidrijsman @nilsbeck ?

Reply:
Hail to the linter! (I guess? 🤷 )

Reply:
I think "unknown word" comes from the spellcheck extension, right? At least, for me that is the case. So, it's not a linter complaint, I think.

Reply:
Oh yeah, that is possible. It was tooling yelling at me 😆